docs/tree-entities.md

```diff
@@ -203,6 +203,9 @@ There are other special methods to work with tree entities through `TreeReposito
 ```typescript
 const treeCategories = await repository.findTrees();
 // returns root categories with sub categories inside
+
+const treeCategories = await repository.findTrees({ relations: ["sites"] });
```

Comment:
I suggest to rename this variable for people who copy/paste from the docs.

Reply:
Good objection!